src/integrationTest/java/uk/gov/hmcts/reform/pip/rules/controllers/ControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.hmcts.reform.pip.rules.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.pip.rules.model.CourtHearings;
+import uk.gov.hmcts.reform.pip.rules.model.Hearing;
+import uk.gov.hmcts.reform.pip.rules.model.Publication;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+public class ControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @DisplayName("Should welcome upon root request with 200 response code")
+    @Test
+    public void welcomeRootEndpoint() throws Exception {
+        MvcResult response = mockMvc.perform(get("/")).andExpect(status().isOk()).andReturn();
+
+        assertThat(response.getResponse().getContentAsString()).startsWith("Welcome");
+    }
+
+    @DisplayName("Should return a 200, and a list containing a single publication, with 1 court and 2 hearings")
+    @Test
+    public void publicationThatExistsTest() throws Exception {
+        MvcResult response = mockMvc.perform(get("/publication/1")).andExpect(status().isOk()).andReturn();
+
+        assertEquals(200, response.getResponse().getStatus(),
+                     "Check that a 200 status is returned is returned");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Publication publication = objectMapper.readValue(
+            response.getResponse().getContentAsByteArray(), Publication.class);
+
+        assertEquals(1, publication.getPublicationId(), "Publication ID is 1");
```

Comment:
Suggestion:
Replace hard-coded value with constant int 1 for both the expected value and the part of the message to avoid typo in message and multiple changes in the future if the expected value to be changed. 

It is the same for all other places similar to this.

Reply:
I've updated to make this a constant, and in the other places I could find